bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -72,15 +77,13 @@ $LocalizationAvailable
         HIGH --> $TimerRunning + name:reorientation
             YES --> @EndTimer + name:reorientation, @StartTimer + name:reorientation_pause + duration:%behavior/body/reorientation_pause_duration
             NO --> $BallSeen
-                YES --> #GoAndKickBallMapGoal
+                YES --> $ClosestToBallNoGoalie
+                    YES --> #GoAndKickBallMapGoal
+                    NO --> @ChangeAction + action:going_to_ball, @GoToBall + distance:1 + target:map_goal
```

Comment:
Why do we also go so close to the ball if we aren't the closest one?

Reply:
We go to the ball with a distance. This is a first simple implementation of supporter.

Reply:
Ah okay